crates/runtime/tests/integration_models.rs

```diff
@@ -44,43 +40,3 @@ fn init_tracing(default_level: Option<&str>) -> DefaultGuard {
         .finish();
     tracing::subscriber::set_default(subscriber)
 }
-
-pub(crate) fn init_tracing_with_task_history(
```

Comment:
Moved to utils so it can be shared between `integration` and `integration_models` tests